bitnami/grafana/values-production.yaml

```diff
@@ -0,0 +1,242 @@
+  customIniConfigMap:
+
+## Create dasboard provider to load dashboards, a default one is created to load
+## dashboards from "/opt/bitnami/grafana/dashboards"
+dashboardsProvider:
+  enabled: false
+  ## ConfigMap with a custom provider file.
+  ## Important to set the Path to "/opt/bitnami/grafana/dashboards"
+  configMapName:
+
+## Create dashboards from a custom configMap that contains the file.
+## They will be mounted by the default dashboard provider if it is enabled
+## Use an array with the configMap names.
+## Example:
+## dashboardsConfigMaps:
+##   - mydashboard
+##   - myotherdashboard
+dashboardsConfigMaps: []
+
+## Create datasources from a custom secret
+## The secret must contain the files
+datasources:
+  secretName:
+
+## Enable persistence using Persistent Volume Claims
+## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+##
+persistence:
+  enabled: true
+  ## wordpress data Persistent Volume Storage Class
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  ##
+  # storageClass: "-"
+  ##
+  ## If you want to reuse an existing claim, you can pass the name of the PVC using
+  ## the existingClaim variable
+  # existingClaim: your-claim
+  accessMode: ReadWriteOnce
+  size: 10Gi
+
+## Configure extra options for liveness and readiness probes
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
+livenessProbe:
+  initialDelaySeconds: 120
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 6
+  successThreshold: 1
+readinessProbe:
+  initialDelaySeconds: 30
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 6
+successThreshold: 1
+
+# Service configuration
+service:
+  port: 3000
+  type: ClusterIP
+  ## Specify the nodePort value for the LoadBalancer and NodePort service types.
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+  ##
+  # nodePort:
+
+  ## Provide any additional annotations which may be required. This can be used to
+  ## set the LoadBalancer service type to internal only.
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+  ##
+  annotations: {}
+
+  ## loadBalancerIP for the PrestaShop Service (optional, cloud specific)
+  ## ref: http://kubernetes.io/docs/user-guide/services/#type-loadbalancer
+  ##
+  # loadBalancerIP:
+
+# Ingress configuration
+ingress:
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: grafana.local
+      paths: ["/"]
+
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+# SecurityContext configuration
+
```

Comment:
Please follow the standard for comments in `values.yaml`. Apply it to the rest of the file.